feature-libs/cart/quick-order/styles/_quick-order-form.scss

```diff
@@ -28,4 +41,69 @@ cx-quick-order-form {
       position: absolute;
     }
   }
+
+  .quick-order-results {
+    background-color: #fff;
+    border-radius: 2px;
+    border: 1px solid var(--cx-color-light);
+    color: #000;
+    display: none;
+    font-size: 13px;
+    position: absolute;
+    top: 47px;
+    width: 300px;
+    // TODO check  for mobiles
+    z-index: 20;
```

Comment:
why 20? Lower number would not work?

Reply:
this is because I decided to use the same z-index like it is in main search on the page